statusHandler/view/termuic/termuiConsole.go

```diff
@@ -1,13 +1,16 @@
 package termuic
 
 import (
+	"errors"
+	"github.com/ElrondNetwork/elrond-go/statusHandler"
+	"github.com/prometheus/common/log"
```

Comment:
should not use this package for logging
Also unsorted imports

Reply:
Done. Logger was changed and imports was sorted.